apps/crn-server/src/autogenerated-gql/graphql.ts

```diff
@@ -2025,22 +2048,33 @@ export type DiscoverDataDto = {
 export type DiscoverDataInputDto = {
   aboutUs: Maybe<DiscoverDataAboutUsInputDto>;
   members: Maybe<DiscoverDataMembersInputDto>;
+  membersTeam: Maybe<DiscoverDataMembersTeamInputDto>;
   pages: Maybe<DiscoverDataPagesInputDto>;
   scientificAdvisoryBoard: Maybe<DiscoverDataScientificAdvisoryBoardInputDto>;
   training: Maybe<DiscoverDataTrainingInputDto>;
   workingGroups: Maybe<DiscoverDataWorkingGroupsInputDto>;
 };
 
-/** The structure of the ASAP Team field of the Discover ASAP content type. */
+/** The structure of the ASAP Team Members field of the Discover ASAP content type. */
 export type DiscoverDataMembersDto = {
   iv: Maybe<Array<Users>>;
 };
 
-/** The structure of the ASAP Team field of the Discover ASAP content input type. */
+/** The structure of the ASAP Team Members field of the Discover ASAP content input type. */
 export type DiscoverDataMembersInputDto = {
   iv: Maybe<Array<Scalars['String']>>;
 };
 
+/** The structure of the ASAP Team field of the Discover ASAP content type. */
+export type DiscoverDataMembersTeamDto = {
+  iv: Maybe<Array<Teams>>;
```

Comment:
What's `iv` mean here? Just `input value`?

Reply:
So this comes from squidex and I assume so. These types are generated from the squidex GraphQL schema. I wouldn't try and review this!